### PR TITLE
Rename to-be-deleted files during remake-shortucts script, actually implemented remaking separated part audio shortcut

### PR DESCRIPTION
**Pull request by demarcod1**

### CODE REVIEW

File: scripts/remake_shortcuts.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def remake_shortcuts():
 
     sep_ids = util.get_separated_folders(service, library_id)
     if sep_ids == None: return 1
+    rm.rename_old_folders(sep_ids)
```
Comment (dhing1024) [inline scripts/remake_shortcuts.py:29]: This function requires two arguments

**Review by dhing1024 — CHANGES_REQUESTED** [review]

File: scripts/remake_shortcuts.py (hunk 1 above)
Comment (demarcod1) [reply scripts/remake_shortcuts.py]: resolved

File: scripts/util/remake_utils.py [hunk 2]
```diff
@@ -74,6 +75,34 @@ def write_shortcuts(service, chartname, id, age, new_folders, alias_map):
...
+        if part == None:
+            print(f'WARNING: Audio file "{partfile_name}" has no matching part folder')
+            continue
+
+        # Make the shortcut
+        util.make_shortcut(service, audiofile_name, audiofile.get('id'), new_folders['aud'][age][part])
+
+# Changes the name of all the old folders to 
+def rename_old_folders(service, sep_ids):
+    for abbr, full in [('old', 'Old'), ('curr', 'Current'), ('fut', 'Future')]:
```
Comment (dhing1024) [inline scripts/util/remake_utils.py:99]: Change `'fut'` to `'future'`

File: scripts/util/remake_utils.py [hunk 3]
```diff
@@ -74,6 +75,34 @@ def write_shortcuts(service, chartname, id, age, new_folders, alias_map):
...
+        for loc in ['sec', 'aud', 'sib']:
+            try:
+                service.files().update(fileId=sep_ids[f'{abbr}_{loc}'],
```
Comment (dhing1024) [inline scripts/util/remake_utils.py:102]: `service.files().update(fileId=sep_ids[f'{loc}_{abbr}'], body={'name': f'{full} Chartz (To be Deleted)'}).execute()`

**Review by dhing1024 — CHANGES_REQUESTED** [review]

**Pull request merged**